### DISCUSSION TITLE
GitHub Achievements

### DISCUSSION THREAD

**Discussion by dkp-consult** [community/community#125614] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hi,

I’ve earned some GitHub Achievements, but I can’t reorganize them. I think it would be nice if I had the choice of position for each achievement.

Regards,

-dkp

**Comment by Wilsmac** [comment]
Saludos <3
Sigue así.

**Comment by VaiBrahma** [comment, marked as answer]
As of now, GitHub does not provide a built-in feature to reorganize achievement badges on user profiles. The badges are automatically arranged by GitHub based on their own criteria, and users do not have control over their order or placement.

However, you can express your interest in this feature by providing feedback to GitHub through their [community forum](https://github.community/) or by submitting a request via their [support page](https://support.github.com/contact). The more users who express interest in this feature, the more likely it is that GitHub will consider implementing it in future updates.
